src/components/CategoryHeader.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Router from "react-router";
+import mui from "material-ui";
+
+export default class CategoryHeader extends React.Component {
+
+    render(): React.Element {
+        return (
+            <div className="CategoryHeader">
+                <div className="found">
+                    I found 25 housing services for men
+                    ages 25-54 in Brunswick
+                </div>
+                <a className="search-link" href="#">
+                    Try a different search
+                </a>
+            </div>
+        );
+    }
```

Comment:
Oops. Someone should debug that.

On Friday, 21 August 2015, Daniel Heath <notifications@github.com> wrote:

> In src/components/CategoryHeader.js
> <https://github.com/ask-izzy/ask-izzy/pull/68#discussion_r37605824>:
>
> > +
> > +export default class CategoryHeader extends React.Component {
> > +
> > +    render(): React.Element {
> > +        return (
> > +            <div className="CategoryHeader">
> > +                <div className="found">
> > +                    I found 25 housing services for men
> > +                    ages 25-54 in Brunswick
> > +                </div>
> > +                <a className="search-link" href="#">
> > +                    Try a different search
> > +                </a>
> > +            </div>
> > +        );
> > +    }
>
> this.state.meta is just a huge URL string
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ask-izzy/ask-izzy/pull/68/files#r37605824>.
>